packages/app/src/AppNav.tsx

```diff
@@ -592,58 +665,90 @@ export default function AppNav({ fixed = false }: { fixed?: boolean }) {
                 />
               )}
             </div>
-            {isSearchExpanded && !isCollapsed && (
-              <>
-                <div className="fw-bold text-light fs-8 ms-3 mt-3">
-                  SAVED SEARCHES
-                </div>
-                {(logViews ?? []).length === 0 ? (
-                  <div className="text-muted ms-3 mt-2">No saved searches</div>
-                ) : null}
-                {(logViews ?? []).map(lv => (
-                  <Link
-                    href={`/search/${lv._id}?${new URLSearchParams(
-                      timeRangeQuery.from != -1 && timeRangeQuery.to != -1
-                        ? {
-                            from: timeRangeQuery.from.toString(),
-                            to: timeRangeQuery.to.toString(),
-                            tq: inputTimeQuery,
-                          }
-                        : {},
-                    ).toString()}`}
-                    key={lv._id}
-                  >
+            {!isCollapsed && isSearchExpanded && (
+              <div className={styles.list}>
+                {isLogViewsLoading ? (
+                  <Loader
+                    color="gray.7"
+                    variant="dots"
+                    mx="md"
+                    my="xs"
+                    size="sm"
+                  />
+                ) : logViews.length === 0 ? (
+                  <Link href="/search">
                     <a
                       className={cx(
-                        'd-flex justify-content-between ms-3 mt-2 cursor-pointer text-decoration-none',
-                        {
-                          'text-success fw-bold':
-                            lv._id === query.savedSearchId,
-                          'text-muted-hover': lv._id !== query.savedSearchId,
-                        },
+                        styles.listLink,
+                        pathname.includes('search') &&
+                          Object.keys(query).length === 0 &&
+                          styles.listLinkActive,
                       )}
-                      title={lv.name}
                     >
-                      <div className="d-inline-block text-truncate">
-                        {lv.name}
+                      <div className="mt-1 ">
```

Comment:
Just dup'ing the same message in chat, not totally sold on using `Live Tail` for empty state, I think ideally we still try to hint to the user as much as we can that they can create saved searches.

Reply:
sounds good, added the empty states back